keyboards/linworks/fave87h/fave87h.c

```diff
@@ -34,15 +34,28 @@ led_config_t g_led_config = {
         {  58,  0 }, {  72,  0 }, {  93,  0 }, { 107,  0 }, { 121,  0 }, { 135,  0 }, { 156,  0 }, { 170,  0 }, { 184,  0 }, { 198,  0 },
         { 216,  0 }, { 230,  0 }, { 244,  0 }, { 244, 64 }, { 230, 64 }, { 216, 64 }, { 197, 64 }, { 179, 64 }, { 162, 64 }, { 144, 64 },
         {  91, 64 }, {  39, 64 }, {  21, 64 }, {   4, 64 }, {  11, 52 }, {  33, 52 }, {  47, 52 }, {  62, 52 }, {  76, 52 }, {  90, 52 },
-        { 104, 52 }, { 118, 52 }, { 132, 52 }, { 146, 52 }, { 160, 52 }, { 186, 52 }, { 230, 52 }
+        { 104, 52 }, { 118, 52 }, { 132, 52 }, { 146, 52 }, { 160, 52 }, { 186, 52 }, { 230, 52 },
+
+        { 230, 48 }, { 230, 36 }, { 230, 24 }, 
+        { 230, 12 }, { 217, 12 }, { 204, 12 }, { 191, 12 }, { 178, 12 }, { 165, 12 }, { 152, 12 }, { 139, 12 }, { 126, 12 }, { 113, 12 }, { 100, 12 }, { 87, 12 }, { 74, 12 }, { 61, 12 }, { 48, 12 }, { 35, 12 }, { 22, 12 },
+        { 14, 24 }, { 14, 36 }, { 14, 48 }, 
+        { 22, 60 }, { 35, 60 }, { 48, 60 }, { 61, 60 }, { 74, 60 }, { 87, 60 }, { 100, 60 }, { 113, 60 }, { 126, 60 }, { 139, 60 }, { 152, 60 }, { 165, 60 }, { 178, 60 }, { 191, 60 }, { 204, 60 }, { 217, 60 }, { 230, 60 }
+
     }, {
         // LED Index to Flag
-        1,    4, 4, 4, 4, 1, 1, 1, 1, 4, 4, 4, 4, 1, 1, 1,
-        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1, 1, 1, 1,
-        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1, 1, 1, 1,
-        9, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,    1,
-        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,    1,       4,
-        1, 1, 1,       4,          1, 1,    1, 1, 1, 1, 1
+        4, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+        1, 1, 4, 4, 1, 1, 1, 1, 1, 1,
+        1, 1, 1, 1, 1, 1, 4, 1, 1, 1,
+        1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+        1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+        1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+        1, 1, 1, 1, 1, 1, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 1, 1, 1, 1, 1,
+        1, 1, 1, 1, 1, 4, 1,
+        1, 1, 1,
+        1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 
+        1, 1, 1,
+        1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1
```

Comment:
Virtually everything being set a a modifier (`1`) is incorrect.

Reply:
Updated from 1 to 2.